COLLABORATOR_GUIDE.md

```diff
@@ -263,6 +263,24 @@ multiple commits. Commit metadata and the reason for the revert should be
 appended. Commit message rules about line length and subsystem can be ignored.
 A Pull Request should be raised and approved like any other change.
 
+### Introducing New Modules
+
+Semver-minor commits that introduce new core modules should be treated with
+extra care.
+
+The name of the new core module should not conflict with any existing 
+module in the ecosystem unless a written agreement with the owner of those
+modules is reached to transfer ownership.
+
+New core modules must be landed with a [Stability Index][] of Experimental.
+Pull requests proposing new core modules must still be marked semver-minor.
+New core modules must remain Experimental until a semver-major release.
+
+It is recommended to give PRs introducing new core modules at least one week
+for review. For new modules that involve significant effort, non-trivial
+additions to Node.js or significant new capabilities, an [EPS][] is
```

Comment:
teeny tiny nit: EPS is plural, EP is the singular ;)

Reply:
heh... sigh. Should I just change it to "Enhancement Proposal"?